proposal.rst

```diff
@@ -268,8 +294,64 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
-
+**Community bonding** (4 MAY-31 MAY)
+ Week 1(4 MAY-10 MAY)
+     Going through weight files again in full detail and discussing doubts on IRC.
+ Week 2(11 MAY-17 MAY)
+     Understanding the writing of Automated test cases.
+ Week 3(18 MAY-24 MAY)
+     Going through Xapian Evaluation files in detail.
+ Week 4(25 MAY-31 MAY)
+     Time for any other discussion needed prior to coding.
+  
+ **Coding**
+  Week 1(1 JUN-6 JUN)
+      - implement Entropy and Global frequency IDF (1 day)
+      - write test cases for Entropy and Global frequency IDF (3 days)
+      - make PR for these changes and getting it reviewed and completing documentation simultaneously (1 day)
```

Comment:
Even though you have some experience with our PR process, I'd suggest leaving more time for the first PR merge of the project. If everything goes more smoothly, you can replan at one of the two "midterm" evaluations.

You talk about documentation here (and in the other subprojects), but it'd be good to understand whether you're talking about API docs, the user guide, or something else. If you want to show how to use some of these new weighting schemes in the user guide, there'll likely be some more work so it might be worth having a subproject to do a number of them in one go.

Reply:
I have added more time for PR merge as suggested. If even more time is required,I would be happy to utilize the first weekend for first PR merge.
I have updated that I am talking about user guide. In this my aim is to advice users which normalisation they should use depending on their dataset.
If I make the documentation a subproject ,then the time I have in the PR merge process won't be utilized.That is the only reason I am doing documentation in patches.

Reply:
It's important to understand that getting a PR merged — particularly the first one — requires not just time from you, but from us. So you being able to work weekends doesn't help unless we happen to have time then, which isn't always going to happen.